pheweb/serve/server_jeeves.py

```diff
@@ -227,30 +250,42 @@ def add_annotations(self, chr, start, end, datalist):
                 d['data']['varid'].append(varid)
                 try:
                     a = annot_hash[varid]['annot']
-                    d['data']['most_severe'].append(a['most_severe'].replace('_', ' '))
-                    d['data']['AF'].append(a['AF'])
-                    d['data']['INFO'].append(a['INFO'])
+                    if 'most_severe' in a:
```

Comment:
could you write lambda that does the checking and returns missing value in case does not exist so no long if/elses are needed

Reply:
I don't know how lambda helps, let me know